tools/per_test_timeouts.py

```diff
@@ -9,6 +9,7 @@
 """
 PER_TEST_TIMEOUTS = {
     "test_environments.py": 1200,  # This test runs through all the environments for 100 steps each
+    "test_environment_determinism.py": 1200,  # This test runs through many the environments for 600 steps each
```

Comment:
Maybe we should reduce the number of steps / environments for this test? This is 20 mins which is quite a bit meaning CI will be 50% slower

I don't see a reason why some environments would be less deterministic than others - wonder if we can just test that both the ManagerBased and Direct workflows are correctly deterministic

Reply:
The current list kinda goes over the different environments that implement some sort of varying PhysX behaviors -- floating base, fixed base, rough terrain, articulations, tendons etc. The idea was to keep this as a minimal set to check that the PhysX solver remains deterministic across these different types of simulations. I reduced now the number of steps to 100.